src/Microsoft.AspNetCore.OData/Builder/ODataConventionModelBuilder.cs

```diff
@@ -55,14 +65,42 @@ public ODataConventionModelBuilder(IServiceProvider provider)
         /// the functionality is due to the way assembly resolution is done in AspNet vs AspnetCore.
         /// </remarks>
         public ODataConventionModelBuilder(IServiceProvider provider, bool isQueryCompositionMode)
+            : this(provider, null, isQueryCompositionMode)
+        {
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="ODataConventionModelBuilder"/> class.
+        /// </summary>
+        /// <param name="provider">The service provider to use.</param>
+        /// <param name="applicationPartManager">
+        /// The application part manager to use. If null, the service
+        /// provider will be queried for the application part manager.
+        /// </param>
+        /// <param name="isQueryCompositionMode">If the model is being built for only querying.</param>
+        /// <remarks>The model built if <paramref name="isQueryCompositionMode"/> is <c>true</c> has more relaxed
```

Comment:
nit regarding consistency: the remarks tag is inline vs. the second remarks tag which is not.